### ISSUE TITLE
[Proposal] Bid view should not write players list

### ISSUE THREAD

**Issue by fabianofrancesconi** [hunterjm/futgui#22]
This is just a proposal but I believe that the values issued by the user are the most important one. Therefore, whenever there is an update for a player happening in the bid frame due to the automatic price retrieval, should not be persisted.

If you also agree, I could do the change myself and add another feature I am working on :)

**Comment by hunterjm** [comment]
I would probably have disagreed when we were using the 15 minute watch / FUTBIN pricing updates, but now that we have our own lowest BIN finder, I think we don't need to persist the values on the bid frame, since they just get overwritten if you choose to use the "Auto Update" feature anyway.

I wouldn't mind pulling it out.  Does anybody else have an opinion?

**Comment by aftershock9** [comment]
I am struggling to understand this proposal? Please can you provide an example?

**Comment by fabianofrancesconi** [comment]
Sure. When you add a player to the list, you manually specify the price for selling and buying.
After you start the auto-bidder, if you have enabled the specific option, the price gets automatically inferred from the current auctions/trades. What happens is that the values you have put get overwritten by the new values. What I am proposing is to actually don't overwrite these values so that, the next time you load the application, it will remain with the prices you chose in the first place.

The feature I am working on, instead, is to give the user the option to enable/disable specific players from being purchased (at the moment the only way to do that is to delete a player from the list).

**Comment by aftershock9** [comment]
Oh okay - I understand the proposal. I generally never set prices and always go with the auto-update feature. I have been burnt in the past by setting prices manually :).

Regarding your feature - what is the value by disabling/enabling specific players? If you have a player that you have disabled, what purpose would that serve?

**Comment by fabianofrancesconi** [comment]
The purpose would be to re-enable that player in the future. At the moment, you could only delete him (and maybe take a note of his value on Evernote or something), and then re-add it if needed. Disabling/Enabling players will allow you to have some sort of "history".

**Comment by aftershock9** [comment]
Oh okay - I understand now. I have been testing and deleting/adding so many players the past few weeks that I wouldn't have used this feature. I can understand the usefulness when you have a core set of players that you trade with and enabling/disabling them during certain periods would make sense e.g. if you were buying/selling giroud at 1000 (auto-price) but then his price shoots up because he scored a hattrick, I would disable him until his price stabilised again.

**Comment by fabianofrancesconi** [comment]
Yes something like that. My goal is to improve a bit the UI with some marginal features as, at the moment, the core code is too much entwined in Jason's mind :p (just kidding).

My goal is to have a finer tuning on the players to sell, which one to update and improve a bit the awareness of the user on what is happening behind.

**Comment by hunterjm** [comment]
I personally think it would be more useful to have the ability to import/export player lists from/to anywhere on the computer.  An automatic copy of the current active list is still saved to `.config`, but that would give users the ability to create multiple lists, and share them across computers.

If we do that, we don't need to worry about "pausing" certain players.

**Comment by shuohan27** [comment]
We can import/export manually by backing up the player config files already.  I would also prefer the autoupdate to overwrite any user specified values if autoupdate is turned on.  However, it would be nice to be able to select whether to autoupdate by player, so instead of a global autoupdate checkmark, have a checkmark by each player to select whether his price is autoupdated or not.  That way I could have the AB doing its usual job while I have it set to grab one or two players which I want at a particular price.
